notifier.cpp

```diff
@@ -102,7 +103,8 @@ int main(int argc, char** argv) {
         // if (s == NATS_OK) {
         // s = stanConnection_PublishAsync(sc, subj, pubMsg->payload, pubMsg->size, _pubAckHandler, (void*)pubMsg);
         for (int i = 0; i < 24 * 1000; ++i) {
-            s = stanConnection_Publish(sc, subj, line.c_str(), line.size());
+            s = stanConnection_PublishAsync(sc, subj, line->c_str(), line->size(), _publish_ack_cb, static_cast<void*>(line));
```

Comment:
do not remove `sync` strategy completely, there should be switch (at least ifdef) to change strategy